### PR TITLE
Bump src-cli minimum version

### PR DESCRIPTION
**Pull request by courier-new**
- Updated minimum version to 4.1.1
- Ran `go generate` for docs

## Test plan

Just version + docs changes.

### CODE REVIEW

**Review by eseliger — APPROVED** [review]

**Review by Piszmog — APPROVED** [review]

**Review by BolajiOlajide — APPROVED** [review]

File: doc/cli/references/extsvc/create.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+| `-name` | exact name of the external service to create |  |
+| `-trace` | Log the trace ID for requests. See https://docs.sourcegraph.com/admin/observability/tracing | `false` |
+| `-user-agent-telemetry` | Include the operating system and architecture in the User-Agent sent with requests to Sourcegraph | `true` |
+
+
+## Usage
+
+```
+Usage of 'src extsvc create':
+  -dump-requests
```
Comment (BolajiOlajide) [inline doc/cli/references/extsvc/create.md:21]: These changes are unrelated. Are they intentional
